Database/Corrections/Automatic/sodBaseItems.lua

```diff
@@ -2827,7 +2827,7 @@ function SeasonOfDiscovery:LoadBaseItems()
             [itemKeys.objectDrops] = nil,
             [itemKeys.itemDrops] = nil,
             [itemKeys.vendors] = nil,
-            [itemKeys.startQuest] = nil,
+            [itemKeys.startQuest] = {78270},
```

Comment:
We don't want to manually edit the `sodBase<X>.lua` files, because we generate them using the scraper and do a full replacement when re-scraping.

Manual overrides should go to the `sod<X>Fixes.lua` - `sodItemFixes.lua` in this case.

Reply:
Okay. I need to go and take another look at the scraper stuff then. So we should avoid doing anything to `sodBaseQuests.lua` as well, and should just be put in to the corresponding `sod<X>Fixes.lua` (`sodQuestFixes.lua` in this case) file?

Fixed in: https://github.com/Questie/Questie/pull/5339/commits/8c03e4eada7c08ed1ad57770512b7205eddf4164

Reply:
In the base files we don't want to do specific fixes for certain quests/fields, because they might get removed/replaced in a later scrape. For the `sodBaseQuests.lua` file in specific we want to add scraped quests one step at a time, to verify they come in with the required fixes. The other base files have already been replaced entirely multiple times.

Therefore manual overrides should go to the corresponding Fixes file.